### ISSUE TITLE
Empty columns in BoM

### ISSUE THREAD

**Issue by APEbbers** [APEbbers/BillOfMaterials-WB#91]

**Comment by gianarb** [comment]
Hello

Thank you for opening this issue, just to double-check is this related to my report here?

https://github.com/APEbbers/BillOfMaterials-WB/pull/90#issuecomment-2207288690

Thanks

**Comment by APEbbers** [comment]
Hi,

Yes indeed. When you select a part in the assembly and click on "add or remove columns", you can actually add the dimension columns. However there are also empty columns which shouldn't be there. Because of that, the dimension columns are outside the screen.

**Comment by gianarb** [comment]
The workaround is correct I was able to add width, height and length. They looked correct, for some reason I got an unexpected `0.12` at the end of almost every one of those measurements that I am not able to figure out from where they are coming from but I am not an expert so maybe it is something at my end since I don't see it reported anywhere but if it sounds something useful to report let me know and I will try

**Comment by APEbbers** [comment]
Hi,

I've fixed the issue with the empty columns. I've tested with your files. I didn't get the extra 0.12.  It might be related to FreeCAD it self, or a small rotation in the assembly. I'm reading the values from FreeCAD internal directly. I'm not calculating my self.
See the picture below with the values i've got:
![image](https://github.com/user-attachments/assets/c88fec5f-a722-42e7-bebb-52316137ffdb)

kind regards
